Source/WTF/wtf/PlatformHave.h

```diff
@@ -1485,3 +1485,7 @@
     || (PLATFORM(WATCHOS) && __WATCH_OS_VERSION_MIN_REQUIRED >= 100000)
 #define HAVE_CTFONTCOPYCOLORGLYPHCOVERAGE 1
 #endif
+
+#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000)
```

Comment:
What about other Apple platforms?

Reply:
> What about other Apple platforms?

Ah, good point! I initially thought we'd keep this change macOS only, but it is good to align the platforms. Fixed in latest patch.

Thanks for reviewing!